remotesigning/validator.go

```diff
@@ -36,6 +36,16 @@ func (v MultiValidator) ShouldSign(webhookEvent webhooks.WebhookEvent) bool {
 	return true
 }
 
+func isL1WalletSigningJob(job SigningJob) bool {
+	return strings.HasPrefix(job.DerivationPath, "m/84")
+}
+
+func isForceClosureClaimSigningJob(job SigningJob) bool {
```

Comment:
We should maybe add the type in the webhook data 😂

Reply:
like add another EventType?

Reply:
not saying we should do it here, but it will be much cleaner. could be a future improvement.

Reply:
ah gotcha. Though they'd still need to validate the derivation path regardless of what we send as that type right? since the backend could lie to them about which type it is if they only validated based on that field?

Reply:
Yeah good point. I think we should probably keep it as is then.